src/parser/esprima_cpp/esprima.cpp

```diff
@@ -368,7 +353,7 @@ class Parser : public gc {
         config.comment = false;
         config.parseSingleFunction = false;
         config.parseSingleFunctionTarget = nullptr;
-        config.parseSingleFunctionChildIndex = SmallValue((uint32_t)0);
+        config.parseSingleFunctionChildIndex = 0;
         /*
         this->config = {
```

Comment:
Remove inactive codes so as to make it clean

Reply:
This code piece is originated from `esprima` parser which is the base code of our parser. It is inactive now, but maybe necessary in the future. So, I think that these esprima codes need to be remained for the ease of implementation later. When ES6 features are completely updated, then we can clean up the unnecessary codes.